### PR TITLE
build: fix wrong indent caused compile warning

### PR DESCRIPTION
**Pull request by gaowanlong**

### CODE REVIEW

File: src/kv/RocksDBStore.cc [hunk 1]
```diff
@@ -692,14 +692,14 @@ bool RocksDBStore::RocksDBWholeSpaceIteratorImpl::valid()
...
 {
   if (valid())
-  dbiter->Next();
+    dbiter->Next();
   return dbiter->status().ok() ? 0 : -1;
 }
 int RocksDBStore::RocksDBWholeSpaceIteratorImpl::prev()
 {
   if (valid())
     dbiter->Prev();
```
Comment (theanalyst) [inline src/kv/RocksDBStore.cc]: can you add {} around the if as well.. so that we're explicit

Comment (gaowanlong) [inline src/kv/RocksDBStore.cc]: @theanalyst yeah, updated now.

**Comment by gaowanlong** [comment]
@theanalyst anyone can merge such really simple pr quickly?

**Comment by gaowanlong** [comment]
@liewegas

**Pull request merged**